index.html

```diff
@@ -24,7 +24,7 @@
             <p id="img-caption">
               Dr. Norman Borlaug, third from the left, trains biologists in
               Mexico on how to increase wheat yields - part of his life-long war
-              on hunger.
```

Comment:
#It would be better to avoid repetition in your code. 
Don't Repeat Yourself (DRY) is an important principle in software development. Code repetition harms maintainability by making it difficult to keep behavior consistent across the application